packages/react/src/MegaMenu/MegaMenu.tsx

```diff
@@ -20,4 +20,5 @@ const MegaMenuRoot = forwardRef(
 
 MegaMenuRoot.displayName = 'MegaMenu'
 
+/** @deprecated Use child components in Header instead. */
```

Comment:
Do we want this deprecation to appear in the change log? If so, should we list it as a ‘feature’?